tests/test_elements/test_mol2any/test_mol2bool.py

```diff
@@ -0,0 +1,24 @@
+"""Test mol to bool conversion."""
+
+import unittest
+
+from molpipeline.abstract_pipeline_elements.core import InvalidInstance
+from molpipeline.mol2any.mol2bool import MolToBool
+
+
+class TestMolToBool(unittest.TestCase):
+    """Unittest for MolToBool."""
+
+    def test_bool_conversion(self) -> None:
+        """Test if the invalid instances are converted to bool."""
+
+        mol2bool = MolToBool()
+        result = mol2bool.transform(
```

Comment:
The element is being called MolToBool - however, here, the input are ints. So generally, it can be any input right? I am not sure whether there are other pipeline elements, that behave liket this? Because it would be an AnyToAny Element?

Reply:
It would be AnyToBool?

Reply:
I would prefer MolToBool to keep the name consistent with the others.